schematic/utils/schema_utils.py

```diff
@@ -6,10 +6,14 @@
 import logging
 import os
 import string
-from typing import Literal, Optional, Union
+from typing import Literal, Optional, Union, Any, TYPE_CHECKING
+import pandas as pd
 
 import inflection
 
+if TYPE_CHECKING:
```

Comment:
Is this because DataModelRelationships is very slow to load? Is there benchmark for the time difference of doing: 
```
if TYPE_CHECKING:
      from schematic.schemas.data_model_relationships import DataModelRelationships
```
versus simply importing by doing: 
```
from schematic.schemas.data_model_relationships import DataModelRelationships
```